src/arviz_plots/plots/lm_plot.py

```diff
@@ -269,20 +289,20 @@ def plot_lm(
     aes_by_visuals.setdefault(
         "central_line", plot_collection.aes_set.difference({"alpha", "color"})
     )
-    if isinstance(ci_prob, Sequence):
-        aes_by_visuals.setdefault("ci_line", {"alpha"})
+    if isinstance(ci_prob, (list | tuple | np.ndarray)):
```

Comment:
I see this check in several places. As far as I understand, you use it to act different in the presence of multiple intervals instead of one.

Would it make sense to convert `ci_prob` to a one-dimensional numpy array early on, and then always act as if they are many? Perhaps something custom can be done if `len(arr) == 1`.